src/elements/ElementAssert.js

```diff
@@ -13,7 +13,6 @@ export default class ElementAssert {
      */
     constructor(elements: Array<Element>) {
         this._elements = elements;
-        this._currentElement = null;
```

Comment:
Do we have any other flow failures? I remember i have those a lot, when i tried to fix this line